server/middlewares/errors.js

```diff
@@ -0,0 +1,28 @@
+/**
+ *  Error Handling Middleware
+ * @param {object} err
+ * @param {object} req
+ * @param {object} res
+ * @param {function} next
+ * @returns {object} Containing Error Message
+ */
+const errorHandler = (err, req, res, next) => {
+  res.status(500).json({
+    status: 'error',
+    message: err.message
+  });
+};
+
+/**
+ * Error Handler for unknown routes
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} Containing Error Message
+ */
+const error404 = (req, res) => {
+  res.status(404).json({
+    message: 'Oops!! Page not found'
+  });
+};
+
+export { errorHandler, error404 };
```

Comment:
This file should also content success method that can be used all over the application and also the file name can be renamed e.g `statusResponse` or `response`

Reply:
@Xwebyna Implemented these feature.